### PR TITLE
framer: update technologies

### PR DESCRIPTION
**Pull request by kurtextrem**
The current patterns do still work, but I wanted to add a few more just in case (the patterns are "or" and not "and" to match a technology, right?)

### CODE REVIEW

**Review by tunetheweb — APPROVED** [review]
LGTM. Yes the patterns are “OR”.

File: src/technologies/f.json [hunk 1]
```diff
@@ -1687,7 +1687,10 @@
     "cats": [
       51
     ],
-    "description": "Framer is primarily a design and prototyping tool. It allows you to design interactive prototypes of websites and applications using production components and real data.",
+    "description": "Framer is a no-code web design platform for designing and publishing responsive websites.",
+    "headers": {
+      "server": "^Framer/",
```
Comment (tunetheweb) [inline src/technologies/f.json:1692]:
```suggestion
      "server": "^Framer/"
```

File: src/technologies/f.json [hunk 2]
```diff
@@ -1704,7 +1707,8 @@
     ],
     "saas": true,
     "scriptSrc": "framerusercontent\\.com",
-    "website": "https://framer.com/sites"
+    "generator": "^Framer",
```
Comment (tunetheweb) [inline src/technologies/f.json:1710]: Is this supports to be in a `meta` property?

Also can you sort the properties?

**Review by tunetheweb — CHANGES_REQUESTED** [review]

File: src/technologies/f.json (hunk 2 above)
Comment (kurtextrem) [reply src/technologies/f.json]: Yep, was meant to be in a `meta` prop. fixed.

Sorting: done!

File: src/technologies/f.json [hunk 3]
```diff
@@ -1696,6 +1699,9 @@
       "Framer.version": "([\\d\\.]+)\\;version:\\1\\;confidence:0",
       "__framer_importFromPackage": ""
     },
+    "meta": {
+      "generator": "^Framer",
```
Comment (tunetheweb) [inline src/technologies/f.json:1703]:
```suggestion
      "generator": "^Framer"
```

File: src/technologies/f.json [hunk 4]
```diff
@@ -1696,6 +1699,9 @@
...
+    }
```
Comment (tunetheweb) [inline src/technologies/f.json:1704]:
```suggestion
    },
```

**Review by tunetheweb — APPROVED** [review]

**Pull request merged**